File: problem-sets/problem-set-1/mario-1/mario.c
#include "cs50.h"
#include <stdio.h>

int main(void)
{
    // Promt user for input
    int height;
    do
    {
        height = get_int("Height: ");
    } 
    // Set condition - height must be a positive integer between 1-8. If not, loop will be repeated
    while (height < 1 || height > 8);
    
    // Right aligned pyramid with dots    
    for (int i = 1; i <= height; i++)
    {
        
        for (int y = height; y > i; y--)
        {
            printf(" ");
        }
        for (int j = 0; j < i; j++) 
        {
            printf("#");
        }
        printf("\n");
    }   
}

File: problem-sets/problem-set-6/sentimental-mario/mario.py
from cs50 import get_int


while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break
for i in range(n):
    for j in range(n - 1 - i):
        print(' ', end='')
    for k in range(i + 1):
        print('x', end='')
    print()




